waagent

```diff
@@ -5909,8 +5916,14 @@ def main():
         else:
             args.append(a)
     global Config
-    Config = ConfigurationProvider()
+    Config = ConfigurationProvider(conf_file)
     
+    logfile = Config.get("Logs.File")
+    if logfile is not None:
+        myLogger.file_path = logfile
+    logconsole = Config.get("Logs.Console")
+    if logconsole is not None and logconsole.lower().startswith("y"):
```

Comment:
Again, it is set by the config. And if the user tries to run waagent from a command line, the serial console is unnecessary if the log is already written to stdout.